### DISCUSSION TITLE
Storage Permission Issue

### DISCUSSION THREAD

**Discussion by moisish** [serversideup/docker-php#498] [Q&A]
I am using the images on AWS ECS for some time now on couple of projects and they work great 

Recently I started playing with docker swarm but I have permission issues when I mount the laravel storage dirs. This makes sense since the image runs as a diff user from the host. Any directions on how people solve issues like this? 

```
The stream or file "/var/www/html/storage/logs/laravel-2024-11-27.log" could not be opened in append mode: Failed to open stream: Permission denied
```

**Comment by jaydrogers** [comment]
Check your `/var/lib/docker/volumes/<volume_name>/_data` folder. Make sure the file permissions match the UID and GID of the `www-data` user in the container (defaults are `33:33` for Debian images and `82:82` for Alpine

**Reply by sawirricardo** [reply]
@jaydrogers in synology nas, we need to set permission on the host folder  to 'Everyone', having read/write permission
